evolver/calibration/standard/polyfit.py

```diff
@@ -0,0 +1,59 @@
+from typing import Self
+
+import numpy.polynomial.polynomial as poly
+from pydantic import Field, model_validator
+
+from evolver.base import ConfigDescriptor
+from evolver.calibration.interface import Calibrator, Transformer
+
+
+class PolyFitTransformer(Transformer):
+    class Config(Transformer.Config):
+        degree: int = Field(ge=0, description="Polynomial degree.")
+        coefficients: list[float] = Field(min_length=1, description="Polynomial coefficients.")
+
+        @model_validator(mode="after")
+        def check_coefficients_length(self) -> Self:
+            if len(self.coefficients) - 1 != self.degree:
+                raise ValueError(f"Degree={self.degree} but {len(self.coefficients)} coefficients given.")
+            return self
```

Comment:
it doesn't really appear degree from config is used at all, or does this somehow get fed from config params into the `fit` function? I might just be missing something, or degree from config is not consumed outside this check?

Reply:
Thanks, good catch. I missed the ``refit`` overrides needed when I converted ``fit`` to a ``classmethod``.